client/src/components/molecules/AvatarBlock.vue

```diff
@@ -5,9 +5,11 @@
       class="q-mb-sm"
     />
     <div class="text-weight-bold">
-      {{ user.name }}
+      <span data-cy="avatar_name">{{ user.name }}</span>
+    </div>
+    <div>
+      <span data-cy="avatar_username">@{{ user.username }}</span>
     </div>
```

Comment:
```suggestion
    <div data-cy="avatar_username">
      @{{ user.username }}
    </div>
```
Same suggestion for the div above, just can't create a suggestion b/c of GitHub limitations.

Reply:
I did this for tighter control of whitespace. Extra spaces are prepended and appended to this element's text if we separate the text into its own line, and that can affect tests that assert an element's text matching something. I realize this can be mitigated with `trim()` or just using methods like Cypress's `contains()` method, but I prefer to have a cleaner DOM to work with.